.sdkmanrc

```diff
@@ -0,0 +1 @@
+java=11.0.20-tem
```

Comment:
to match the version used in [build and release workflows](https://github.com/search?q=repo%3Aprovenance-io%2Fp8e-cee-api+java-version&type=code)